doc/code_snippets/test/performance/compression_speed.lua

```diff
@@ -0,0 +1,112 @@
+-- This script is used to estimate the speed of CRUD operations
+-- for different compression algorithms: zstd, lz4, and zlib.
+-- See also: https://www.tarantool.io/en/enterprise_doc/tuple_compression/
+
+box.cfg {
+    memtx_memory = 2 * 1024 * 1024 * 1024,
+    log_level = 1,
+    wal_mode = none
+}
```

Comment:
Another important thing - `wal_mode = none` does not disable WAL! It should be `wal_mode = 'none'`.
After WAL is disabled, do not forget to set fiber slice to a large value.
And, why have you decided to disable WAL for benchmark?

Reply:
> And, why have you decided to disable WAL for benchmark?

Sorry, don't have a good answer :) For some reason, I thought it will reduce the load on the VM.

Reply:
Enabled WAL will be a bottleneck for a single-fiber benchmark. So it must be disabled for this one.

Reply:
ok, thanks!